.github/workflows/build.yml

```diff
@@ -532,7 +532,7 @@ jobs:
         uses: actions/setup-dotnet@v3
 
       - name: Run Tests
-        run: dotnet test --filter '${{ matrix.config.filter}}' --configuration release --logger trx --blame --collect:"XPlat Code Coverage" --settings ./.github/workflows/codecov.runsettings --results-directory ./TestResults/
```

Comment:
>trx [](http://example.com/codeflow?start=96&length=3)

I also noticed you took out the `--logger trx` option. Are trx files being dropped to the test results directory? The task that follows this one uploads trx files as a build artifact in case they are needed.

Reply:
Never mind. I see the logger settings in runsettings.